src/OrchardCore.Modules/OrchardCore.Workflows/Http/Scripting/HttpMethodsProvider.cs

```diff
@@ -118,6 +118,10 @@ public HttpMethodsProvider(IHttpContextAccessor httpContextAccessor)
                             result = values.ToArray();
                         }
                     }
+                    else if (httpContextAccessor.HttpContext.Request.Form.Count > 0)
+                    {
+                        result = httpContextAccessor.HttpContext.Request.Form.ToArray();
```

Comment:
An array of what? internally is an IEnumerable<KeyValue>>. That's why the document states it would return a string instead, which I agree is still broken without any change.

Reply:
It returns an array of the form fields and their values. 

At the time i was trying to use the workflow task for loop to iterate through the form array, but i was not able to get the form array since it just returned null. If you would rather return a text string of all the form values then you can delete the pr.

Reply:
Could you paste an example with what it looks like as an array, and as a string?
We still need to fix it, I am just wondering what is the most appropriate option for the scenario.

Reply:
I'm not sure what you mean with what it looks like? Its just an array of strings from the form. It works and returns in the same way, as the `_queryStringMethod` does it.

What i'm using today however, is requestFormAsJson and then new Array(json.MyArray) when i need to iterate. But im sure there are still cases where iterating form values could be useful or/and easier for some users.